Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -15,10 +15,12 @@ import Mathlib.Tactic.IntervalCases
 Given `SMul G X`, an action of a type `G` on a type `X`, we define
 
 - the predicate `IsBlock G B` states that `B : Set X` is a block,
-  which means that the sets `g • B`, for `g ∈ G`, are equal or disjoint.
+which means that the sets `g • B`, for `g ∈ G`, are equal or disjoint.
+Under `Group G` and `MulAction G X`, this is equivalent to the classical
+definition `MulAction.IsBlock.def_one`
```

Comment:
As indicated by the highlighting within vscode, you are supposed to indent after a bullet point
```suggestion
  which means that the sets `g • B`, for `g ∈ G`, are equal or disjoint.
  Under `Group G` and `MulAction G X`, this is equivalent to the classical
  definition `MulAction.IsBlock.def_one`
```